code/modules/mob/living/simple_animal/borer.dm

```diff
@@ -91,9 +91,15 @@ var/global/borer_unlock_types_leg = typesof(/datum/unlockable/borer/leg) - /datu
 	// Event handles
 	var/eh_emote
 
-/mob/living/simple_animal/borer/New(var/loc)
+	var/list/name_prefixes = list("Primary","Secondary","Tertiary","Quaternary","Quinary","Senary","Septenary","Octonary","Nonary","Denary")
```

Comment:
this list should be static, so that there's only one list for ALL borers, as opposed to 10 lists (of identical contents) for 10 borers.